.vscode/settings.json

```diff
@@ -19,5 +19,6 @@
         },
         "MD034": false,
         "MD038": false
-    }
+    },
+    "powershell.codeFormatting.addWhitespaceAroundPipe": true
```

Comment:
Ah, undo this change.  That particular setting has caused issues depending on the version of the PS extension for VSCode (and version of PSSA).  This should also remove this file from the PR as it should have no changes.

Reply:
Done.